### PR TITLE
Exegol-images 2.1.0

### PR DESCRIPTION
**Pull request by ShutdownRepo**
- switching from kali linux base to debian base (more stable, cleaner, lighter, etc.)
- adding anew, naabu, username-anarchy, Starkiller, dfscoerce, coercer, tls-scanner, kerbrute
- fixing httpx, eyewitness, GHunt, checksec, robotstester, feroxbuster, Empire, 
- improving BloodHound queries
- hardening install (umask) and improving environment and zsh configuration
- ARM builds now possible (dockerhub builds yet to be added, ARM builds are only possible locally for now)
- and a few other things I guess

### CODE REVIEW

**Comment by Dramelac** [comment]
With the change of the base from kali to debian, shouldn't the version be changed to 3.0.0?

**Comment by ShutdownRepo** [comment]
Maybe 2.1.0 instead of 2.0.3, but I'd like to have 3.0.0 for when Exegol images will have optimized layers, better installs (e.g. custom repository with packaged tools), and CI/CD (basic system & tools checks)
What do you think?

**Comment by Dramelac** [comment]
2.1.0 sounds good for me

**Comment by Dramelac** [comment]
and maybe precise in the description of the release (not mandatory on the PR) but explicit the switch from kali to debian it's a major change for this release

**Comment by ShutdownRepo** [comment]
> and maybe precise in the description of the release (not mandatory on the PR) but explicit the switch from kali to debian it's a major change for this release

100% agree! I forgot, editing the PR title and description

File: sources/install.sh [hunk 1]
```diff
@@ -10,16 +10,33 @@ function colorecho () {
...
+
+function criticalecho-noexit () {
+  echo -e "${RED}[EXEGOL ERROR] $@${NOCOLOR}" 2>&1
+}
+
 function update() {
   colorecho "Updating, upgrading, cleaning"
-  apt-get -y update && apt-get -y install apt-utils && apt-get -y upgrade && apt-get -y autoremove && apt-get clean
+  echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+  apt -y update && apt -y install apt-utils dialog && apt -y upgrade && apt -y autoremove && apt-get clean
```
[the remark this answers is not in the document]
Comment (Dramelac) [reply sources/install.sh:25]: and not just here, just a search find more than 40 usage of `apt`.
Is it intended ?

[the remark this answers is not in the document]
Comment (ShutdownRepo) [reply sources/install.sh:25]: Nope, almost certain it's not intended, probably a leftover of the kali -> debian change. These need to be changed.

[the remark this answers is not in the document]
Comment (ShutdownRepo) [reply sources/install.sh]: Should be ok now, let's wait for the dockerhub build to finish successfully (just in case) before merging

**Review by Dramelac — APPROVED** [review]

**Pull request merged**